### PR TITLE
CI: Use the old TM policy

### PR DESCRIPTION
**Pull request by Tal-or**
We didn't bump the RTE u/s version yet, so we need to keep the old TM policy

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

File: hack/run-e2e-kni.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ PROJECT_DIR="${BASE_DIR}"/..
 
 RTE_NAMESPACE="${RTE_NAMESPACE:-rte-e2e}"
 E2E_NAMESPACE_NAME="${E2E_NAMESPACE_NAME:-rte-e2e}"
-E2E_TOPOLOGY_MANAGER_POLICY="${E2E_TOPOLOGY_MANAGER_POLICY:-SingleNUMANodeContainerLevel}"
+E2E_TOPOLOGY_MANAGER_POLICY="${E2E_TOPOLOGY_MANAGER_POLICY:-single-numa-node}"
```
Comment (fromanirh) [inline hack/run-e2e-kni.sh:13]: ok, CI passed because is not using the new scripts yet, right?

Comment (Tal-or) [reply hack/run-e2e-kni.sh:13]: Indeed.

Comment (fromanirh) [reply hack/run-e2e-kni.sh:13]: works for me.

**Review by fromanirh — APPROVED** [review]
/lgtm

**Pull request merged**